src/scripts/choices.js

```diff
@@ -1572,47 +1573,55 @@ class Choices {
     this._wasTap = true;
   }
 
+  /**
+   * Handles mousedown event in capture mode for containetOuter.element
+   * @param {MouseEvent} event
+   */
   _onMouseDown(event) {
-    const { target, shiftKey } = event;
-    // If we have our mouse down on the scrollbar and are on IE11...
-    if (
-      this.choiceList.element.contains(target) &&
-      isIE11(navigator.userAgent)
-    ) {
-      this._isScrollingOnIe = true;
+    const { target } = event;
+    if (!(target instanceof HTMLElement)) {
+      return;
     }
 
-    if (
-      !this.containerOuter.element.contains(target) ||
-      target === this.input.element
-    ) {
+    // If we have our mouse down on the scrollbar and are on IE11...
+    if (IS_IE11 && this.choiceList.element.contains(target)) {
+      // check if click was on a scrollbar area
+      const firstChoice = /** @type {HTMLElement} */ (this.choiceList.element
```

Comment:
Can we put the comment above the variable please :) then you don't need to do the braces:

```js
/** @type {HTMLElement} */
const firstChoice =  this.choiceList.element.firstElementChild;
```

Reply:
Like we do here: https://github.com/jshjohnson/Choices/blob/master/src/scripts/choices.js#L53-L54

Reply:
it's better to give a type to `choiceList.element`, so, VSCode will pick it up here. Do you want me to do that as part of this PR?